prow/plugins/releasenote/releasenote.go

```diff
@@ -74,8 +75,16 @@ var (
 )
 
 func init() {
-	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment, nil)
-	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, nil)
+	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment, helpProvider)
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	return &pluginhelp.PluginHelp{
+			Description: "The releasenote plugin implements a release note process that uses a markdown 'releasenote' code block to associate a release note with a pull request. Until the 'releasenote' block in the pull request body is populated the PR will be assigned the '" + releaseNoteLabelNeeded + "' label.\nThere are three valid types of release notes that can replace this label:\n\t- PRs with a normal release note in the 'releasenote' block are given the label '" + releaseNote + "'.\n\t- PRs that have a release note of 'none' in the block are given the label '" + releaseNoteNone + "' to indicate that the PR does not warrant a release note.\n\t- PRs that contain 'action required' in their 'releasenote' block are given the label '" + releaseNoteActionRequired + "' to indicate that the PR introduces potentially breaking changes that necessitate user action before upgrading to the release.\n\nTo support old behavior, this plugin also provides a '/release-note-none' command that can be used by organization members to specify that no release note is needed for the PR as an alternative to setting the 'releasenote' block contents to 'none'.",
```

Comment:
nit: can we use back-tick strings and format this with newlines embedded so it's a little easier to read in the code?

Reply:
Done.